src/test/java/util/xcmailr/XcMailrApiTest.java

```diff
@@ -1,58 +1,16 @@
 package util.xcmailr;
 
 import java.io.IOException;
-import java.util.Base64;
-import java.util.UUID;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import javax.mail.MessagingException;
-
-import org.aeonbits.owner.ConfigFactory;
-import org.apache.http.client.ClientProtocolException;
-import org.junit.After;
 import org.junit.Assert;
-import org.junit.BeforeClass;
 import org.junit.Test;
 
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
-
-import util.xcmailr.data.EmailAccount;
-import util.xcmailr.util.Credentials;
-import util.xcmailr.util.SendEmail;
 import util.xcmailr.util.SendRequest;
 
-public class XcMailrApiTest extends AbstractTest
+public class XcMailrApiTest extends AbstractXcMailrApiTest
```

Comment:
Rename this class to something more meaningful. Something along the line of `EmailCreationApiTest` may be a good one.

Reply:
You renamed `AbstractXcMailrApiTest`. Why did you opt to rename the abstract class instead of the one I wanted to change?